contracts/sophia.md

```diff
@@ -513,7 +513,7 @@ The block-chain environment available to a contract is defined in three name spa
 
 ### Exceptions
 
-Contracts can fail with an (uncatchable) exception using the function
+Contracts can fail with an (uncatchable) exception using the built-in function
 
 ```
 abort(reason : string) : 'a
```

Comment:
What does `'a` mean? Maybe it shall be [tuple of zero elements](https://github.com/aeternity/epoch/pull/1623/files#diff-188efb931c56a6fb0aa1263174fc8eabR108)?

Reply:
That I don't know, it was already there. It is a type variable so it could be what I want it to be.  As far as I know there is no `no_return` type which would be the best.  Having it as `()` would be better and match the current implementation. I will fix it.

Reply:
Fixed it so docs say it return `()`.